client/process.go

```diff
@@ -136,7 +136,8 @@ func (config *Config) DebugGCS() {
 		cmd += debugCommand("ls -l /tmp/gcs/*")
 		cmd += debugCommand("cat /tmp/gcs/*/config.json")
 		cmd += debugCommand("ls -lR /var/run/gcsrunc")
```

Comment:
This should probably change too now.... (I don't think /var/run/gcs* is used anymore)

Reply:
It still is used. We write the pid file there for create container/process

Reply:
It isn't used for logs anymore, though.

Reply:
I agree but isn't it usefully to get an output of all the pid files? IE: all the running process in runc?

Reply:
Sure. The more the merrier :)

Reply:
Alternatively I can move the use of this dir to be /tmp/gcs/(id)/ too?

Reply:
If we don't care about the runc wrapper being a generic wrapper that can be used outside of this project, I think it makes sense to merge it as much as possible with the GCS. Also, I guess in addition to pid files there are also process.json files under /var/run/gcsrunc, which could definitely be useful for logging.

Reply:
@beweedon - Want to open an issue for merging the two then? I don't see any use cases for a runc wrapper library. Its primarily just for us and how we use runc

Reply:
Sure thing!

Reply:
#144